elasticsearch_dsl/search.py

```diff
@@ -313,6 +317,30 @@ def fields(self, fields=None):
         s._fields = fields
         return s
 
+    def partial_fields(self, **partial):
+        """
+        Control which part of the fields to extract from the `_source` document
+
+        :arg partial: dict specifying which fields to extract from the source
```

Comment:
this is not a single arg, but kwargs instead, could we have examples of what the method call will look, instead of what the dict should be?